src/index.js

```diff
@@ -2,7 +2,7 @@ const { exec } = require('child_process')
 const execPromise = require('util').promisify(exec)
 
 /* MAC PLAY COMMAND */
-const macPlayCommand = path => `afplay ${path}`
+const macPlayCommand = (path, volume) => `afplay ${path} -v ${volume}`
```

Comment:
Might want to escape that path for macOS, in case of file paths with whitespace. Ex:

`const macPlayCommand = (path, volume) => `afplay \"${path}\" -v ${volume}``

Reply:
Indeed. Thanks for your review !